core/runtime.rs

```diff
@@ -634,6 +634,8 @@ impl JsRuntime {
       state.waker.register(cx.waker());
     }
 
+    self.pump_v8_message_loop();
```

Comment:
I had to move this to before (some of) the module loading because if a module is top-level-awaiting for wasm compilation, it might end up not being loaded yet but with no pending ops immediately after the compilation is done. I guess the same applies to ops which might be started from the microtasks after pumping the message loop, but it's not something I've run into while testing.

Reply:
This looks ok 👍  I wasn't sure where to put it, but if you confirmed that it needs to be triggered before then it's okay